tests/src/test/scala/system/basic/WskCliBasicTests.scala

```diff
@@ -451,7 +451,7 @@ class WskCliBasicTests extends TestHelpers with WskTestHelpers {
     getJSONFromResponse(trigger.stdout, true).fields("parameters") shouldBe JsArray(
       JsObject("key" -> JsString("a"), "value" -> JsString("A")))
     getJSONFromResponse(trigger.stdout, true).fields("publish") shouldBe false.toJson
-    getJSONFromResponse(trigger.stdout, true).fields("version") shouldBe "0.0.2".toJson
+    getJSONFromResponse(trigger.stdout, true).fields("version") shouldBe "0.0.1".toJson
```

Comment:
this is semantic changing - it suggests there was a call to trigger update somewhere before that is no longer occurring.

in fact on line 438 and 439 you see there was a `create` and then an `update`. So I think this change is not correct and should be reverted.

Reply:
Nice catch!